Frameworks/UIKit/UIButton.mm

```diff
@@ -552,8 +552,7 @@ - (void)setTitleColor:(UIColor*)color forState:(UIControlState)state {
     // ConvertUIColorToWUColor:nil creates a valid WUColor with null comObj
     // which isn't what we want
     if (color) {
-        WUColor* convertedColor = XamlUtilities::ConvertUIColorToWUColor(color);
-        WUXMSolidColorBrush* titleColorBrush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
+        WUXMSolidColorBrush* titleColorBrush = [[WUXMSolidColorBrush makeInstanceWithColor:XamlUtilities::ConvertUIColorToWUColor(color)] autorelease];
```

Comment:
Awesome! Thanks for pushing this!

---
In reply to: [97875917](https://github.com/Microsoft/WinObjC/pull/1803#discussion_r97875917) [](ancestors = 97875917,97843932)